cclib/method/ddec.py

```diff
@@ -437,10 +492,9 @@ def condition_densities(self):
                 self._candidates_phi[atomi].append(phiAI[atomi])
                 self._candidates_bigPhi[atomi].append(bigphiAI[atomi])
 
-
             # lowerbigPhi is largest negative Phi.
             # upperbigPhi is smallest positive Phi.
-            if fitphi:
+            if not phiconverged:
```

Comment:
Is it possible for phiconverged to be True at this point?

Reply:
This was possible before I made some changes on the while loop. I must've forgotten to account for the change. Thanks for noting this!